addons/sale_timesheet/models/project.py

```diff
@@ -200,15 +200,13 @@ def _compute_allocated_hours(self):
         uoms_dict = {uom.id: uom for uom in self.env['uom.uom'].browse(uom_ids)}
 
         for project in self:
-            if not project.sale_line_id:
-                project.allocated_hours = 0
-                continue
```

Comment:
I don't understand why you remove this code. 🤔

Reply:
@xavierbol , moved code and added the reason in the commit message. Thanks